### ISSUE TITLE
airodump-ng show spaces instead wpa2 encrypt

### ISSUE THREAD

**Issue by NicholasFalcon** [aircrack-ng/aircrack-ng#2098]
# Issue type

- Defect - Incorrect value displayed/received/stored

# System information

- OS:
Linux hironori 4.19.0-6-amd64 #1 SMP Debian 4.19.67-2+deb10u1 (2019-09-20) x86_64 GNU/Linux
- CPU:
Intel(R) Core(TM) i5-9400F CPU @ 2.90GHz
- Wireless card and chipset (if relevant):
ASUSTek Computer, Inc. USB-AC56 802.11a/b/g/n/ac Wireless Adapter [Realtek RTL8812AU]
Realtek Semiconductor Corp. RTL8812AU 802.11a/b/g/n/ac 2T2R DB WLAN Adapter

# Aircrack-ng version

- Version:
Aircrack-ng 1.5.2

- Commit Revision hash:
rev 4e79066a

# Defect
## `ap_cur->security` value is incorrect
![screenairodump-ng](https://user-images.githubusercontent.com/42118570/69336352-5aa6b780-0c6f-11ea-8830-0e244d38b847.png)

airodump-ng/dump_write.c (206)
```
if ((ap_cur->security & (STD_OPN | STD_WEP | STD_WPA | STD_WPA2)) == 0)
    fprintf(opt.f_txt, " ");
else
{
    if (ap_cur->security & STD_WPA2)
    {
        if (ap_cur->security & AUTH_SAE || ap_cur->security & AUTH_OWE)
            fprintf(opt.f_txt, " WPA3");
        fprintf(opt.f_txt, " WPA2");
    }
    if (ap_cur->security & STD_WPA) fprintf(opt.f_txt, " WPA");
    if (ap_cur->security & STD_WEP) fprintf(opt.f_txt, " WEP");
    if (ap_cur->security & STD_OPN) fprintf(opt.f_txt, " OPN");
}
```

the lines above are written to the file, but the protocol is not defined in the file either

`ap_cur->security` is equal to `9376` in all lines in the screenshot

commands before launch airodump-ng:
```
ip link set
ip link set <interface> up
ip link set <interface> down
iw dev <interface> set type monitor
ip link set <interface> up
airodump-ng <interface>
```

**Comment by RockyBukake** [comment]
### Same issue but on **Aircrack 1.6**
![2020-02-09_23-32](https://user-images.githubusercontent.com/35773324/74112424-6fbb2480-4b94-11ea-83b6-b4264d394195.png)
**OS** : Arch Linux (5.5.2-arch1-1)
**CPU**: Intel i5-7300HQ (4) @ 3.500GHz
**Wireless card** : ALFA AWUS036NHA 2.4Ghz
### **I temporarily fixed it by downgrading to 1.5.2 and works just fine ...**

**Comment by aircrack-ng** [comment]
Fixed in e0e9e1e8b6f3dac863c3c46dcef4848ce57d46a6 and added test aad65178d1b2a645a7b7faae625a97d16d443293
